sqx_test.go

```diff
@@ -99,49 +93,50 @@ func newWidget(status string) Widget {
 }
 
 func TestRead(t *testing.T) {
-	setupTestWidgetsTable(t)
+	tx := Tx(t)
+	setupTestWidgetsTable(t, tx)
 	dbWidget := newDBWidget()
```

Comment:
Any thoughts on refactoring `newDBWidget` to take the `tx` arg? Then you'd want to hoist all of the `tx := Tx(t)` and `setupTestWidgetsTable(t, tx)` into the individual `t.Run` calls, but it would illustrate how we typically create our dbModel handle at the point at which we want to run a set of queries in a transaction.

Reply:
I think that's a stytch-ism and might not represent common use. At other places I've been, usually services/dao layers/repositories are created on application startup, not ephemerally per request. 

That being said, I can do either way

Reply:
Sounds good. I have no strong feelings either way